zend_abstract_interface/interceptor/php8/interceptor.c

```diff
@@ -536,32 +538,61 @@ void zai_interceptor_replace_observer(zend_op_array *op_array, bool remove) {
         *endHandler = handlers.end;
     }
 }
+#else
+#define ZEND_OBSERVER_DATA(function) \
```

Comment:
I won't pretend to know what's going on in the interceptor here. I won't block the merge on it, but maybe we could talk about this some time tomorrow or next week.

Reply:
Sounds like a plan, we definitely can talk a bit about that.